x-pack/metricbeat/module/azure/storage/integration_tests/config/main.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/core"
+	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/storage"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+const (
+	resourceGroupName = "observability-beats-test-storage-account"
+	location          = "WestEurope"
+	storageAccount    = "storageobsaccount"
```

Comment:
We have to ensure that the same scenario can be started multiple times. This is needed to avoid conflicts when running the same tests in the same account.
If there cannot be two storage accounts with the same name, we have to ensure that unique names are used.
The [random provider](https://www.pulumi.com/docs/intro/cloud-providers/random/) probably can help here, we do also something like that with terraform: https://github.com/elastic/beats/blob/2151d157819b40b7a70b6deeab5aa46af25b57b6/x-pack/metricbeat/module/aws/terraform.tf#L35

Reply:
it is already doing that, the resource group will start with that name and it adds an unique identifier to it , same with storage account. We do not need to configure anything on our side.

Reply:
Oh, this is great then.